packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -260,7 +265,12 @@ export default {
           }
           // useEffectEvent() return value is always unstable.
           return true;
-        } else if (name === 'useState' || name === 'useReducer') {
+        } else if (
+          name === 'useState' ||
+          name === 'useReducer' ||
+          name === 'useFormState' ||
+          name === 'useActionState'
```

Comment:
Would we want to shorten this to - 
```
['useState', 'useReducer', 'useFormState', 'useActionState'].includes(name)
```

Reply:
One less array allocation and can be better optimized by runtimes.